.make.defaults

```diff
@@ -234,6 +234,7 @@ __check_defined = \
 	mkdir ${LIB_NAME}
 	cp -p -R ${LIB_PATH}/src ${LIB_NAME}
 	cp -p -R ${LIB_PATH}/pyproject.toml ${LIB_NAME}
+	-cp -p -R ${LIB_PATH}/requirements.txt ${LIB_NAME}
```

Comment:
this looks redundant with the if 2 lines down.

Reply:
done.